js/ui/text_box/ui.text_editor.base.js

```diff
@@ -331,6 +334,14 @@ const TextEditorBase = Editor.inherit({
         });
     },
 
+    _compressInputContainer: function() {
+        const $element = this.$element();
+        const isIE11 = browser.msie && browser.version <= 11;
+        if(isIE11 && $element.css('display') === 'block') {
```

Comment:
Is it works fine with the SSR?
The method is used in the `initMarkup` function, so the test, which checks this behavior, should be in "_markup" file and pass in SSR environment.
Or it should be called after `_initMarkup`.